odoo/orm/models.py

```diff
@@ -1218,17 +1226,14 @@ def _validate_fields(self, field_names, excluded_names=()):
                 check(records)
 
     @api.model
-    def default_get(self, fields_list):
-        """ default_get(fields_list) -> default_values
-
-        Return default values for the fields in ``fields_list``. Default
+    def default_get(self, fields_list: Sequence[str]) -> ValuesType:
```

Comment:
Please be aware that those methods are autodocumented in the documentation.

![image](https://github.com/user-attachments/assets/b3713602-4e19-4bb6-9f9a-ebb8486c45e0)
becomes 
![image](https://github.com/user-attachments/assets/332eb9a3-fdd9-454b-9bb8-318fd8e9a0b8)

`ValuesType` cannot be more abstract and unclear honestly.
Also, is it really conflicting to keep the 'rst-style' types in the docstring?  They are displayed in the documentation and IMO it's meaningful to have the generic type that devs should use as param and expected as return value, in a way more understandable than this complex typing abstraction.

Reply:
I have checked the documentation. If you have any suggestions to rename `ValuesType` we could do that rename. `default_values` was not a meaningful type (for a method that returns default values).
You should not keep rst-style typing if you use python typing. They will diverge and become confusing. In your example, we accept any sequence of str, not arbitrary lists.

Reply:
Thanks for the answer.
I understand we might not want to keep the two typing indeed, but in this case, do we really need to 'factorize' dumb typing values ? I've no magic idea of better name for `ValuesType`, but is it really necessary to factorize this instead of 'simply' specifying `dict[str, typing.Any]`.

Reply:
It is still quite common (> 30 usages).